src/Frontend/src/views/throughputreport/transport.ts

```diff
@@ -7,4 +7,5 @@ export enum Transport {
   RabbitMQ = "RabbitMQ",
   SQLServer = "SqlTransport",
   AmazonSQS = "AmazonSQS",
+  PostgreSQL = "PostgreSql",
```

Comment:
Nitpick: The spelling seems to be "PostgreSQL" (https://www.postgresql.org/) should we align with that?

Reply:
We do - the left hand side is the enum value and the right hand side is whats coming from SC so we are using that just to match.

Reply:
Ah, should we adjust the sc side in that case to be consistent?

Reply:
It's internal only - in line with the project name.
I don't mind. Sql transport does something similar where the name of project etc uses Sql, then it shows externally as SQL So in the postgres naming we followed that pattern.
I'm fine either way.

Reply:
Ok, fair enough 👍